edx/cs_comments_service
# SE-3175 Bump codecov to latest version

Codecov 0.1.x was yanked from rubygems, causing installation of
cs_comments_service to fail. This bumps to the latest codecov version from rubygems.org.


**Jira tickets**: [OSPR-4904](https://openedx.atlassian.net/browse/OSPR-4904)

**Merge Deadline**: ASAP - this is breaking provisioning for all Open edX instances that use cs_comments_service.

**Test instructions**:

- verify that everything that uses codecov (eg. tests that upload code coverage data) still work
- verify that installation succeeds.

**Author notes**:

The [codecov changelog at this revision](https://github.com/codecov/codecov-ruby/blob/04297efd2e844106129f33aa0adf4b9d10604399/CHANGELOG.md) does not mention any breaking changes. The minor version bump appears to be related to which api it uses to upload the data to, rather than changing the local api.

**Reviewers**:


Fixes #322
